### DISCUSSION TITLE
Feature Currency Convertor added

### DISCUSSION THREAD

**Discussion by Sunnyyu33a** [DerpSnepbek/Tafe24S1T112_Ext#1] [Ideas]
Hi Oliver,

My feature branch Currency Convertor has been added, not merged to development yet. Would you like to add your feature branch in before I merge?

**Comment by DerpSnepbek** [comment]
Actually, you can go ahead and merge it if you like. Since I've already got a copy of the repository I'll be fine to merge mine once I'm home.
